configure.py

```diff
@@ -294,6 +302,49 @@ def _ffmpeg_install(cfg):
     return cmds
 
 
+@_block("sdl2-install", [])
+def _sdl2_install(cfg):
+    dirs = (
+        "lib",
+        "include",
+    )
+    cmds = []
+    for d in dirs:
+        src = op.join(cfg.externals["sdl2"], d, "*")
+        dst = op.join(cfg.prefix, d)
+        cmds.append(_cmd_join("xcopy", src, dst, "/s", "/y"))
+
+    src = op.join(cfg.externals["sdl2"], "lib", "x64", "SDL2.dll")
+    dst = op.join(cfg.prefix, "Scripts")
+    cmds.append(_cmd_join("xcopy", src, dst, "/y"))
+
+    pc = textwrap.dedent(
```

Comment:
Can we have a comment explaining the (temporary?) need for the .pc file? Ideally a reference to the meson issue.